Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -393,15 +393,9 @@ + (WSSInMemoryRandomAccessStream*)_populateStream:(WSSInMemoryRandomAccessStream
 }
 
 + (WSSInMemoryRandomAccessStream*)_grabStreamFromUIImage:(UIImage*)image {
-    CGImageRef img = [image CGImage];
-
-    // TODO #1338 - Support via encoded data from IWIC
-    woc::unique_cf<CGImageRef> imgRef(_CGImageCreateCopyWithPixelFormat(img, GUID_WICPixelFormat32bppPBGRA));
-    // TODO #1450 - CGDataProvider should not be an NSData.
-    NSData* data = static_cast<NSData*>(CGImageGetDataProvider(imgRef.get()));
-
     StrongId<WSSInMemoryRandomAccessStream> stream;
     stream.attach([WSSInMemoryRandomAccessStream make]);
+    NSData* data = UIImagePNGRepresentation(image);
```

Comment:
>UIImagePNGRepresentation [](start = 19, length = 24)

in WOCCataglog, there is copy and paste page which contains copying / pasting graphic data. minding test that?